### ISSUE TITLE
Upgrade ArcGIS Enterprise from 10.7. to 10.7.1 - Portal WA Install failed silently

### ISSUE THREAD

**Issue by Biboba** [Esri/arcgis-powershell-dsc#187]
Hello,

I just upgraded from 10.7 to 10.7.1 and though everything seemed to be successful, I could not access the Portal as the wedadaptor was not registered with the Portal. Having a look at the log, I noticed that Portal Web Adaptor registration failed silently:
```
[PORTAL01]: LCM:  [ Start  Set      ]  [[ArcGIS_WebAdaptor]ConfigurePortal-PORTAL01.domain.com]
[PORTAL01]:                            [[ArcGIS_WebAdaptor]ConfigurePortal-PORTAL01.domain.com] https://gisportal.domain.com/geoportal/webadaptor
[PORTAL01]:                            [[ArcGIS_WebAdaptor]ConfigurePortal-PORTAL01.domain.com] Output of execution:- Unable to register with Portal. The remote server returned an error: (500) Internal Server Error.
[PORTAL01]: LCM:  [ End    Set      ]  [[ArcGIS_WebAdaptor]ConfigurePortal-PORTAL01.domain.com]  in 1.2810 seconds.
[PORTAL01]: LCM:  [ End    Resource ]  [[ArcGIS_WebAdaptor]ConfigurePortal-PORTAL01.domain.com]`
```

I ran again the configuration and everything went fine.

Many thanks for this library : it's awesome when everything work :)

Thanks

**Comment by shailesh91** [comment]
@Biboba We have identified the issue affecting registering a Portal Web Adaptor after Portal Upgrade. This is due to a reindexing operation as part of the WA Registration. This call from WA is timing out hence the 500 Error. This shouldn't affect your deployment in any manner if you rerun the upgrade.  
We are working towards implementing a solution for this.

**Comment by Biboba** [comment]
@shailesh91 Thanks for the feedback. Glad to read that you were able to identify the issue.

**Comment by cameronkroeker** [comment]
@Biboba,

This issue has been addressed in the latest release, v3.0.0:

https://github.com/Esri/arcgis-powershell-dsc/releases
